main.py

```diff
@@ -5,18 +5,21 @@
 '''
 List of available commands:
 
-    read_sensor_temperature()
-    read_sensor_humidity()
-    read_sensor_gas()
-    read_sensor_distance()
-    read_sensor_motion()
+    print("*text*")
+    sleep(*seconds*)
```

Comment:
Here, the stars should also be removed. Instead, a good default value should be used.

Better:
`sleep(5)`